app/ldap.go

```diff
@@ -19,11 +19,15 @@ import (
 func (a *App) SyncLdap(includeRemovedMembers bool) {
 	a.Srv().Go(func() {
 
-		if license := a.Srv().License(); license != nil && *license.Features.LDAP && *a.Config().LdapSettings.EnableSync {
-			if ldapI := a.Ldap(); ldapI != nil {
-				ldapI.StartSynchronizeJob(false, includeRemovedMembers)
+		if license := a.Srv().License(); license != nil && *license.Features.LDAP {
+			if *a.Config().LdapSettings.EnableSync {
```

Comment:
Just a minor nit: It is a bit unidiomatic to have nested `else`s in Go. Rather, we match the negative case and return early.  In this case, it would be 
```go
if !a.Config().LdapSettings.EnableSync {
  mlog.Error("...")
  return
}

ldapI := a.Ldap();
if ldapI == nil {
  mlog.Error("...")
  return
}
ldapI.StartSynchronizeJob(false, includeRemovedMembers)
```

But doesn't need to block the PR.

Reply:
Makes total sense, I fixed it. Thanks!